.github/actions/gcp-docker-login/action.yml

```
@@ -17,7 +17,8 @@ runs:
         token_format: access_token
         workload_identity_provider: "projects/397012414171/locations/global/workloadIdentityPools/github-actions-pool/providers/github-actions"
         service_account: "github-actions@github-iam-387915.iam.gserviceaccount.com"
-        export_environment_variables: false
+        export_environment_variables: true
```

Could we see if this works? I don't see any obvious reason we need the env vars exported in the workflow.

```suggestion
        export_environment_variables: false
```